plugins/preparer/client.go

```diff
@@ -0,0 +1,38 @@
+package preparer
+
+import (
+	"errors"
+	"net/rpc"
+)
+
+type rpcClient struct {
+	client *rpc.Client
+}
+
+var _ Preparer = &rpcClient{}
```

Comment:
What's the function of this initialization?

Reply:
Ensures that `rpcClient` adheres correctly to the `Preparer` interface. Not 100% sure it's necessary, but I don't see a place where that check is made implicitly (through arguments or return values) at the moment.